core/metacling/src/TClingDeclInfo.h

```diff
@@ -18,6 +18,7 @@
 #define ROOT_TClingDeclInfo
 
 #include <string>
+#include <clang/AST/Type.h>
```

Comment:
Could you move the clang header before string. We should order includes by generality.

Reply:
Fixed.